Mage/src/main/java/mage/filter/predicate/other/NotManaAbilityPredicate.java

```diff
@@ -0,0 +1,26 @@
+package mage.filter.predicate.other;
+
+import mage.abilities.Ability;
+import mage.filter.predicate.Predicate;
+import mage.game.Game;
+import mage.game.stack.StackObject;
+
+/**
+ * @author notgreat
+ */
+public enum NotManaAbilityPredicate implements Predicate<StackObject> {
+    instance;
+
+    @Override
+    public boolean apply(StackObject input, Game game) {
+        if (!(input instanceof Ability)) {
+            return false;
+        }
+        return !((Ability) input).isManaAbility();
```

Comment:
Looks strange. Mana abilities never goes to stack due mtg rules. So it’s useless to search stack objects for it.

Reply:
Huh, that's a good point. If I understand correctly, that means that `FilterActivatedOrTriggeredAbility` just automatically includes the "not a mana ability" clause by its very design?

Reply:
Yep, before this it was only being used for targeting